Fix integration tests wildfly,h2

**Describe the bug**
The integration tests for Wildfly,h2 are failing. The remaining test failing is `SpinClassloadingTest`.
```
Caused by: org.operaton.spin.xml.SpinXmlDataFormatException: SPIN/DOM-XML-01028 Cannot deserialize '[xmlserializable: null]...' to java class 'org.operaton.bpm.integrationtest.functional.spin.XmlSerializable'
Caused by: jakarta.xml.bind.JAXBException: No JAXBContextFactory implementation found
```

It looks that the spin-xml module does not find the JAXB implementation.

**To Reproduce**
```
./mvnw -DskipTests -Pdistro,distro-webjar,h2-in-memory,wildfly,distro-wildlfy clean install
./mvnw -Pengine-integration,wildfly,h2 clean verify -f qa
./mvnw -Pwildfly,h2,engine-integration -f qa/integration-tests-engine -Dsurefire.includes="**/SpinClassloadingTest.java"
```

**Expected behavior**
All tests are passing

**Additional context**
JAXB is provided by module `org.glassfish.jaxb`

--- a/qa/integration-tests-engine/src/test/java-wildfly/org/operaton/bpm/integrationtest/util/TestContainer.java
+++ b/qa/integration-tests-engine/src/test/java-wildfly/org/operaton/bpm/integrationtest/util/TestContainer.java
@@ -34,6 +34,7 @@ public class TestContainer {
   }
 
   public static void addContainerSpecificResourcesWithoutWeld(WebArchive webArchive) {
+    webArchive.addAsManifestResource("jboss-deployment-structure.xml");
     webArchive.addAsLibraries(DeploymentHelper.getEjbClient());
   }
 

--- a/qa/integration-tests-engine/src/test/resources-wildfly/jboss-deployment-structure.xml
+++ b/qa/integration-tests-engine/src/test/resources-wildfly/jboss-deployment-structure.xml
@@ -4,6 +4,7 @@
     <dependencies>
       <module name="org.operaton.bpm.operaton-engine" />
       <module name="org.mybatis.mybatis"/>
+      <module name="org.glassfish.jaxb" services="import"/>
     </dependencies>
   </deployment>
 </jboss-deployment-structure>

